### DISCUSSION TITLE
Deployment issue

### DISCUSSION THREAD

**Discussion by PhilipKrook** [statiqdev/Discussions#69] [Q&A]
Hi
I'm new to this.
Struggling to deploy a small site I'm building.

It works fine locally. 
It's built with Statiq and I use Netlify to deploy it.
It's somewhat working, it renders a html file, but only that one. I tried using stuff from the statiq-biolerplate demo but mine is not rendering the files as it should.

I have the same folder structure and as an example it does not render my _partials/_head.cshtml file it seems.
Looking in output/ folder the output/index.cshtml does include the content of the _head.cshtml file, but the preview page on Netlify does not.

What am I missing? Thanks

**Comment by daveaglick** [comment]
This sounds like a potential deployment issue - if the expected content is in the file in "output", then Statiq did it's part correctly (at least in terms of generating the output files from Razor and Markdown).

How is the Netlify deployment handled? Are you using the Statiq Netlify deployment support, uploading manually, or building on the Netlify build agents? What does your Netlify site management screen say about the site - does it show the deployment having taken place and does it indicate the site was successfully deployed?

**Reply by PhilipKrook** [reply]
The only thing I can find is this 

" 8:00:55 AM: The build completed successfully, but the following processes were still running:
8:00:55 AM: ​
8:00:55 AM:    - /opt/buildhome/.dotnet/dotnet exec /opt/buildhome/.dotnet/sdk/5.0.202/Roslyn/bincore/VBCSCompiler.dll -pipename:G4PIsWgOWGb+ULGHlVdjzj50ducRuzBjY8VVuiRohr8
8:00:55 AM: Post processing - HTML
8:00:55 AM: ​
8:00:55 AM: These processes have been terminated. In case this creates a problem for your build, refer to this https://answers.netlify.com/t/support-guide-how-to-address-the-warning-message-related-to-terminating-processes-in-builds/35277">article for details about why this process termination happens and how to fix it. "

It could be netlify skips some part(s) of the build when deploying and therefor misses huge parts of the site.
